Makefile

```diff
@@ -30,6 +30,10 @@ secrets:
 	cp vendor/native-secrets/android/koala_endpoint.xml app/src/main/res/values/koala_endpoint.xml \
 		|| cp config/koala_endpoint.xml.example app/src/main/res/values/koala_endpoint.xml
 
+# Copy crashlytics over. Fallback to examples if they don't exist
```

Comment:
let's add `fabric.properties` to our secrets and it over here so that anyone who runs `make secrets` can pull these credentials down